modules/cloud-run-v2/README.md

```diff
@@ -84,6 +85,34 @@ module "cloud_run" {
 # tftest modules=2 resources=4 fixtures=fixtures/secret-credentials.tf inventory=service-volume-secretes.yaml e2e
 ```
 
+## Mounting GCS buckets
+
+```hcl
+module "cloud_run" {
+  source     = "./fabric/modules/cloud-run-v2"
+  project_id = var.project_id
+  name       = "hello"
+  region     = var.region
+  containers = {
+    hello = {
+      image = "us-docker.pkg.dev/cloudrun/container/hello"
+      volume_mounts = {
+        bucket = "/bucket"
+      }
+    }
+  }
+  volumes = {
+    bucket = {
+      gcs = {
+        bucket       = var.bucket
+        is_read_only = false
+      }
+    }
+  }
```

Comment:
FYI this won't quite work as-is: this kind of volume mount requires a V2 execution environment which isn't the default.

Reply:
Thanks for pointing this out